aa_stripe/models.py

```diff
@@ -409,8 +409,15 @@ def charge(self, idempotency_key=None):
             metadata = {
                 "object_id": self.object_id,
                 "content_type_id": self.content_type_id,
+                "member_uuid": str(self.user.uuid),
```

Comment:
This lib is public and some other users might not have this field
```suggestion
                "member_uuid": str(self.user.uuid) if hasattr(self.user, "uuid") else None,
```

Reply:
made the updates